RadixWallet.xcodeproj/project.pbxproj

```diff
@@ -8508,8 +8456,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/tgrapperon/swift-dependencies-additions";
 			requirement = {
-				kind = exactVersion;
-				version = 1.0.1;
+				kind = upToNextMajorVersion;
+				minimumVersion = 1.0.1;
```

Comment:
Unsure about this. We should use **exact** pinning no?

Reply:
This one actually works with exact pinning now, it didn't last week. The other two I had just forgotten to change back after updating.